report/src/presence/co2.tex

```diff
@@ -0,0 +1,14 @@
+\section{Presence detection using $\text{CO}_{2}$ }\label{sec:presence_env}
+$\text{CO}_{2}$ falls under the category of an \textit{environmental system}. 
+These system measure physical attributes of an environment, and reacts according to these measurements.
+These systems can be used to infer occupancy of rooms by detecting changes in environmental parameters caused by human presence.
```

Comment:
```suggestion
Therefore, such systems can be used to infer occupancy of rooms by detecting changes in environmental parameters caused by human presence.
```
"These" twice in a row sounds odd. Maybe my suggestion is not the right one - But consider something along these lines.